code/modules/admin/verbs/adminhelp.dm

```diff
@@ -593,7 +593,7 @@ GLOBAL_DATUM_INIT(ahelp_tickets, /datum/admin_help_tickets, new)
 			continue
 		if(target_servers && !(I in target_servers))
 			continue
-		INVOKE_ASYNC(world, /world/Export, "[servers[I]]?[list2params(message)]")
+		INVOKE_ASYNC(world, "Export", "[servers[I]]?[list2params(message)]")
```

Comment:
I'm unsure if this will work tbh. Have you tested it?

Reply:
It does not in fact work.

It seems to be failing silently.